src/lib/ts-internal.ts

```diff
@@ -98,7 +98,7 @@ export function isBindingPattern(node: ts.Node): node is ts.BindingPattern {
 
 // https://github.com/Microsoft/TypeScript/blob/v2.1.4/src/compiler/utilities.ts#L1729
 export function getClassExtendsHeritageClauseElement(node: ts.ClassLikeDeclaration | ts.InterfaceDeclaration) {
-  return tsany.getClassExtendsHeritageClauseElement.apply(this, arguments);
+  return tsany.getClassExtendsHeritageElement.apply(this, arguments);
```

Comment:
This should probably be a separate PR and the function name should be updated

Reply:
+1 to rename the exported function in a separate PR.

Reply:
@aciccarello Do you think it's better to keep both old and new exported functions for backward compatibility?

Reply:
Since it is only used internally, I'd say change the calling function (I think there is only one) and get rid of any `getClassExtendsHeritageClauseElement` references.

Reply:
This gave me an idea for #830 which is related